src/server/index.ts

```diff
@@ -33,14 +33,32 @@ export const createServerRunConfig = (portOrConfig?: any, callback?: any) => {
   return config;
 };
 
+/**
+ * Wrap a user-provided auth function to simplify external API
+ * @param  {function} fn The authentication function to wrap
+ * @return {function} Wrapped function for use by master
+ */
+const wrapAuthFn = fn => ({ action, gameMetadata, playerID }) =>
+  fn(action.payload.credentials, gameMetadata[playerID]);
```

Comment:
Could you add a test that covers this line?

Reply:
Sure! Might take a moment for me to get to it, but I will.

Reply:
No hurry. Thanks!